PR: Add sqrtpi package (#507)
stdlib-js/stdlib: lib/node_modules/@stdlib/math/base/special/sqrtpi/docs/repl.txt

{{alias}}( x )
    Computes the positive square root of the product of pi and a provided

>     positive number.

Reviewer: ```suggestion
    Computes the principal square root of the product of 𝛑 and a positive
    number.
```

    For `x < 0`, the square root is not defined.

    Parameters
    ----------
    x: number
        Input value.

    Returns
    -------
    y: number

>         Principal square root of the product of 𝛑 and the input number.

Reviewer: ```suggestion
        Principal square root of the product of 𝛑 and the input value.
```

>         Positive square root of the product of pi and the input number.

Reviewer: ```suggestion
        Principal square root of the product of 𝛑 and the input number.
```

    Examples
    --------
    > var y = {{alias}}( 4.0 )

>     3.5449

Reviewer: ```suggestion
    ~3.5449
```

    > y = {{alias}}( 10.0 )

>     5.6049

Reviewer: ```suggestion
    ~5.6049
```
